contracts/vault-managers/CDPManager01.sol

```diff
@@ -111,6 +80,8 @@ contract CDPManager01 is ReentrancyGuard {
             // mint USDP to owner
             vault.borrow(asset, msg.sender, usdpAmount);
 
+            _chargeBorrowFee(asset, usdpAmount);
```

Comment:
It may make sense to add an explicit user parameter to `_chargeBorrowFee` so that they would go hand to hand and be called with the same `(asset, msg.sender, usdpAmount)` args.